packages/ui/src/components/va-stepper/VaStepper.demo.vue

```diff
@@ -0,0 +1,115 @@
+<template>
+  <VbDemo>
+    <VbCard title="Default">
+      <VaStepper
+        v-model="step"
+        :steps="steps"
+      >
+        <template
+          v-for="step, i in steps"
+          :key="step.label"
+          #[`step-content-${i}`]
+        >
+          This is {{ step.label }} content.
+        </template>
+      </VaStepper>
+    </VbCard>
+    <VbCard title="With disabled steps">
+      <VaStepper
+        v-model="step"
+        :steps="stepsWithDisabled"
+      >
+        <template
+          v-for="step, i in stepsWithDisabled"
+          :key="step.label"
+          #[`step-content-${i}`]
+        >
+          This is {{ step.label }} content.
+        </template>
+      </VaStepper>
+    </VbCard>
```

Comment:
I'm able to navigate by changing the v-model. 
<img width="758" alt="image" src="https://user-images.githubusercontent.com/29167241/219011816-5cc613cf-f3ac-43c8-a66c-ecb003eed5db.png">

Reply:
It's a feature.